go/kbfs/libkbfs/prefetcher.go

```diff
@@ -699,7 +711,16 @@ func (p *blockPrefetcher) run(testSyncCh <-chan struct{}) {
 			ptr := ptrInt.(BlockPointer)
 			pre, ok := p.prefetches[ptr.ID]
 			if !ok {
-				p.log.Debug("nothing to cancel for block %s", ptr)
+				p.queuedPrefetchHandlesLock.Lock()
+				ch, ok := p.queuedPrefetchHandles[ptr.ID]
+				if ok {
+					ch <- struct{}{}
```

Comment:
This isn't where we should be doing the canceling.  It should be in the actual `CancelPrefetch` function.  Doing it here doesn't do much good, since the the request could have already been processed after the caller decided to cancel it.